wandb/sdk/launch/utils.py

```diff
@@ -192,6 +193,8 @@ def construct_launch_spec(
         launch_spec["docker"] = {}
     if docker_image:
         launch_spec["docker"]["docker_image"] = docker_image
+    if sweep_id:
+        launch_spec["sweep_id"] = sweep_id
```

Comment:
this is only present in schedulers right? Maybe we could use this to check if an agent popped a sweep?

Reply:
ah, nice

Reply:
Actually, this is only passed in for runs belonging to a sweep, the scheduler gets override args with the sweep_id.